Cargo.toml

```diff
@@ -18,5 +18,5 @@ api_test = []
 
 [dependencies]
 rustc-serialize = "~0.3"
-hyper = "~0.5"
+hyper = "0.6.4"
```

Comment:
Will this force executable dependents to use `0.6.4` specifically? Would it be better to have a more flexible semver spec like `^0.6.4`? The way I understand it, there is a set of symbols for each version of a dependency in the final product.